tests/acceptance/replica_availability.feature

```diff
@@ -0,0 +1,6 @@
+Feature: replica_availability
+
+  Scenario: Calling the replica_availability command on a cluster with unavailable replicas
```

Comment:
I wish we were running kafka on multiple brokers for the test, then we could have relatively easily tested a negative testcase also.